keras_preprocessing/image/image_data_generator.py

```diff
@@ -654,15 +654,18 @@ def flow_from_dataframe(self,
             drop_duplicates=drop_duplicates
         )
 
-    def standardize(self, x):
+    def standardize(self, x, inplace=True):
         """Applies the normalization configuration to a batch of inputs.
 
         # Arguments
             x: Batch of inputs to be normalized.
+            inplace: If to change x in place when doing transformations.
```

Comment:
Rather than doing this, I think we should modify the docstring. See https://github.com/keras-team/keras/pull/10153 which is a very similar PR. The discussion is roughly the same.

Reply:
Yes. I believe the problem is that the documentation is not explicit about it. We should make sure this behavior is documented (and maybe explain why we choose to do the modification in-place)  and also provide the workaround `standardize(np.copy(x))`.

Sometimes a good documentation is all we need.

Reply:
done @gabrieldemarmiesse. Let me know if you agree, and I will close the related issue as well.